android_cobertura.md

```diff
@@ -120,7 +26,7 @@ Add this build step and choose `Use Gradle Wrapper`.
 
 Check `From Root Build Script Dir`.
 
-Under Tasks, please put: `clean testDebugUnitTest unitTestCoverage -Pcoverage assembleQa` **Note:** assembleQa is needed for Northeastern, but might not be needed for your Android project.
+Under Tasks, please put: `clean testDebugUnitTest unitTestCoverage -Pcoverage assembleQa`
```

Comment:
we might want to still mention that some project uses assembleQa and other projects uses assembleDebug (or none if the build is just for running tests and not uploading the resulting apk to the ota)